packages/xo-web/src/common/xo/index.js

```diff
@@ -1572,7 +1572,15 @@ export const deleteTemplates = templates =>
     body: _('templateDeleteModalBody', { templates: templates.length }),
   }).then(async () => {
     const defaultTemplates = []
+    const protectedTemplates = []
     let nErrors = 0
+
+    const showError = () =>
+      error(
+        _('failedToDeleteTemplatesTitle', { nTemplates: nErrors }),
+        _('failedToDeleteTemplatesMessage', { nTemplates: nErrors })
+      )
```

Comment:
Why did you move this function?

Reply:
i tried to move it outside of the promise so ```nErrors``` is only verified in the last callback and ```showErrors``` is called only if ```nErrors``` is not null

Reply:
I'm not sure I understand this, this shouldn't have any impact. Also, you can inline it if it's only used once.